### PR TITLE
Period Keyword Argument instead of n_days

### PR DESCRIPTION
**Pull request by maximilian-gelbrecht**
See #417 

`@deprecate` doesn't support keyword arguments, so I had to do this a bit more manual.

### CODE REVIEW

File: src/run_speedy.jl [hunk 1]
```diff
@@ -2,18 +2,24 @@
 $(TYPEDSIGNATURES)
 Run a SpeedyWeather.jl `simulation`. The `simulation.model` is assumed to be initialized."""
 function run!(  simulation::Simulation;
-                n_days::Real = 10,
-                output::Bool = false)
+                period::Period = Day(10),
```
[the remark this answers is not in the document]
Comment (milankl) [reply src/run_speedy.jl]: addressed with [3bacbde](https://github.com/SpeedyWeather/SpeedyWeather.jl/pull/420/commits/3bacbdea3e64a90b874877f924834c911526f130)

**Comment by maximilian-gelbrecht** [comment]
Ok, great! 

This can be merged, right? 
Does this also close #417? I think we changed all user-facing parameters, or did we forget something?

**Comment by milankl** [comment]
I think we have the all!

**Comment by maximilian-gelbrecht** [comment]
@milankl I just noticed that with README.md changed, people who install the package from the regular Julia registry might be misled with the `period` not working for them. Would you consider registering a v0.7.1. of Speedy that includes this PR? Then everybody gets the same user interface and no confusion about which keyword to use.

**Comment by milankl** [comment]
Happening with https://github.com/SpeedyWeather/SpeedyWeather.jl/pull/422 and https://github.com/JuliaRegistries/General/pull/96657

**Pull request merged**